### PR TITLE
Reports

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

File: communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package org.jboss.da.communication.aprox.impl;
 
+import java.io.FileNotFoundException;
```
Comment (thescouser89) [inline communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java]: import order not respected! (I think)

File: reports-backend/src/main/java/org/jboss/da/reports/backend/api/VersionFinder.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import org.jboss.da.communication.model.GAV;
 
 import java.util.List;
+import org.jboss.da.communication.CommunicationException;
```
Comment (thescouser89) [inline reports-backend/src/main/java/org/jboss/da/reports/backend/api/VersionFinder.java]: import order not respected!

File: reports-backend/src/test/java/org/jboss/da/reports/impl/TestReportsGenerator.java [hunk 3]
```diff
@@ -0,0 +1,220 @@
+package org.jboss.da.reports.impl;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.jboss.da.communication.CommunicationException;
```
Comment (thescouser89) [inline reports-backend/src/test/java/org/jboss/da/reports/impl/TestReportsGenerator.java]: import order!

File: reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java [hunk 4]
```diff
@@ -19,6 +19,7 @@
 
 import com.wordnik.swagger.annotations.Api;
 import com.wordnik.swagger.annotations.ApiOperation;
+import org.jboss.da.communication.CommunicationException;
```
Comment (thescouser89) [inline reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java]: import order!

**Comment by jbartece** [comment]
Rebase to the master and then retest please

**Comment by jbartece** [comment]
@janinko needs rebase

**Pull request merged**